XenModel/XenAPI/HTTP.cs

```diff
@@ -299,6 +357,39 @@ public static Uri BuildUri(string hostname, string path, params object[] args)
             return uri.Uri;
         }
 
+        private static List<string> GetFields(List<string> lines, string field)

```

Comment:
Do we need these new get methods? Why not use linq FindAll, FirstOrDefault etc. instead?